### PR TITLE
Create ECR conference page & update landing page

### PR DESCRIPTION
**Pull request by jwade1327**
Screen Shot 2024-02-13 at 11 39 55 AM

Screen Shot 2024-02-13 at 11 39 39 AM

### CODE REVIEW

**Comment by jwade1327** [comment]
[Setup ECR section - AMEU](https://trello.com/c/fMkIs6Xd/531-setup-ecr-section-ameu)

File: sites/auntminnieeurope.com/server/routes/conference.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+const queryFragment = require('@parameter1/base-cms-marko-web-theme-monorail/graphql/fragments/website-section-page');
+
+const withPinned = require('../templates/website-section/conference-with-pinned');
+const conference = require('../templates/website-section/conference');
+
+const ecrAliases = [
+  '2024',
+  '2023',
+  '2019',
+  '2018',
```
Comment (Shinsina) [inline sites/auntminnieeurope.com/server/routes/conference.js:11]: image


These don't exist, are they suppose to?

**Review by Shinsina — CHANGES_REQUESTED** [review]
I'm also assuming that the templates like `conference-with-pinned` were copy pasted from AuntMinnie, if that's the case they should likely be moved up to the global/shared package and imported accordingly, assuming there is no differences between each site.

File: sites/auntminnieeurope.com/config/page-details.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
+const ecrSiblingRoutes = [
+  { title: 2024, href: 'resources/conferences/ecr/2024', alt: '2024 European Congress of Radiology (ECR) News Coverage' },
+  { title: 2023, href: 'resources/conferences/ecr/2023', alt: '2023 European Congress of Radiology (ECR) News Coverage' },
+  { title: 2019, href: 'resources/conferences/ecr/2019', alt: '2019 European Congress of Radiology (ECR) News Coverage' },
+  { title: 2018, href: 'resources/conferences/ecr/2018', alt: '2018 European Congress of Radiology (ECR) News Coverage' },
```
Comment (Shinsina) [inline sites/auntminnieeurope.com/config/page-details.js:5]: These likewise should be corrected with the sections that actually exist (this goes for the entire file)

**Review by Shinsina — CHANGES_REQUESTED** [review]

**Review by Shinsina — APPROVED** [review]

**Pull request merged**